### ISSUE TITLE
`forge-topic-list-columns` returns `eieio-unbound`

### ISSUE THREAD

**Issue by bkchr** [magit/forge#438]
I have added the following config:

```
  (setq forge-topic-list-columns
      '(("#" 5 forge-topic-list-sort-by-number (:right-align t) number nil)
        ("Title" 60 t nil title  nil)
        ("State" 6 t nil state nil)
        ("Labels" 8 t nil labels nil)
        ("Assignees" 10 t nil assignees nil)
        ("Updated" 10 t nil updated nil)))
```

And then when I list all issues, `labels` and `assigness` always is set to `eieio-unbound`.

**Comment by tarsius** [comment]
The value of these slots always is `eieio-unbound` in the respective table. When using the oo interface as in `(oref (epkg "a-package") labels)`, then the value is looked up in another table, returned and cached in the object.

We don't use the objects here because creating many objects is very slow.

In the short term I don't intend to do anything about this, so you won't be able to display this information in the the `tabulated-list` based lists. I believe that in other packages that use `tabulated-list` I added support for specifying a function instead of a slot name. That function could then requests object and get the value from that or it could query the appropriate table directly.

This would have the known performance penalty. Multiplied by the number of columns for which you do that. So there would have to be some optimization. This is why I don't intend to add support for this any time soon. And once I have figured a satisfactory way of doing it I should use it in all my packages that use this interface ...
